<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">BDSP Randomizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>BDSP.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Pokemon Options</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbGen1">
         <property name="text">
          <string>Gen 1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="cbGen2">
         <property name="text">
          <string>Gen 2</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbGen3">
         <property name="text">
          <string>Gen 3</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbGen4">
         <property name="text">
          <string>Gen 4</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="cbLegends">
         <property name="text">
          <string>Keep Legendarys</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3" alignment="Qt::AlignLeft">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Randomizers</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="cbStarters">
         <property name="text">
          <string>Randomize Starters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbPokemon">
         <property name="text">
          <string>Randomize Pokemon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSafari">
         <property name="text">
          <string>Randomize Safari</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUnderground">
         <property name="text">
          <string>Randomize Underground</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbEvolutions">
         <property name="text">
          <string>Randomize Evolutions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTrainers">
         <property name="text">
          <string>Randomize Trainers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbShops">
         <property name="text">
          <string>Randomize Shops</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTM">
         <property name="text">
          <string>Randomize TMs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTMCompat">
         <property name="text">
          <string>Randomize TM Compat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbAbilities">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Randomize Abilities</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbFieldItems">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Randomize Field Items</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbMoves">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Randomize Moves</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Coming Soon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbLevels">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Randomize Levels</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>141</height>
       </size>
      </property>
      <property name="title">
       <string>Level Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="rbFlat">
         <property name="text">
          <string>Level Multiplier</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="sbMax">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>



Increase%</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QTextEdit" name="tbLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Utilities</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="cb60FPS">
         <property name="text">
          <string>60FPS Mod</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTimeSkip">
         <property name="text">
          <string>Speed Multipler</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Timestep</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Speedup
Amount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sbTimeStep">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>*Lockes</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbIronLocke">
         <property name="font">
          <font>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string notr="true">IronLocke</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbPokemon_3">
         <property name="font">
          <font>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Mega RandomLocke</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSafari_3">
         <property name="font">
          <font>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string notr="true">Nuzlocke</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="btnRandomize">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Randomize</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
